### PR TITLE
ISPN-14880 RESP DBSIZE

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-14880

### CODE REVIEW

File: server/resp/src/main/java/org/infinispan/server/resp/commands/connection/DBSIZE.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * <a href="https://redis.io/commands/dbsize/">DBSIZE</a>
+ *
+ * @since 15.0
+ */
+public class DBSIZE extends RespCommand implements Resp3Command {
+   public DBSIZE() {
+      super(0, 0, 0, 0);
```
Comment (karesti) [inline server/resp/src/main/java/org/infinispan/server/resp/commands/connection/DBSIZE.java:21]: arity is 1

File: server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java [hunk 2]
```diff
@@ -566,4 +567,10 @@ public void testStrlenNotPresent() {
       String key = "strlen-notpresent";
       assertThat(redis.strlen(key)).isEqualTo(0);
    }
+
+   @Test
+   public void testDbSize() {
+      RedisCommands<String, String> redis = redisConnection.sync();
+      assertThat(redis.dbsize()).isEqualTo(cache.size());
```
Comment (karesti) [inline server/resp/src/test/java/org/infinispan/server/resp/RespSingleNodeTest.java:574]: we should test a value set and comparing the size

**Review by karesti — CHANGES_REQUESTED** [review]

**Comment by tristantarrant** [comment]
Fixed the issues

**Review by jabolina — APPROVED** [review]
LGTM, but needs a rebase.

**Comment by tristantarrant** [comment]
Rebased

**Review by wburns — APPROVED** [review]
LGTM

**Review by karesti — APPROVED** [review]

**Pull request merged**